src/BlazorUI/Bit.BlazorUI/Components/Inputs/Dropdown/BitDropdown.scss

```
@@ -623,9 +710,10 @@
 .bit-drp-clr,
 .bit-drp-cai {
     display: flex;
-    right: spacing(3.5);
     width: spacing(3.75);
-    height: spacing(3.75);
+    // Follows the height of the dropdown like Prefix/Suffix do, so the button fills the trigger
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)